tron/serialize/runstate/statemanager.py

```diff
@@ -146,6 +146,8 @@ def restore(self, job_names, skip_validation=False):
             self._restore_metadata()
 
         jobs = self._restore_dicts(runstate.JOB_STATE, job_names)
+        # jobs should be a dictionary that contains  job name and number of runs
+        # {'Master.k8s': {'run_nums':[0], 'enabled': True}, 'Master.cits_test_frequent_1': {'run_nums'; [1,0], 'enabled': True}}
```

Comment:
```suggestion
        # jobs should be a dictionary that contains  job name and number of runs
        # {'MASTER.k8s': {'run_nums':[0], 'enabled': True}, 'MASTER.cits_test_frequent_1': {'run_nums': [1,0], 'enabled': True}}
```

just to make this more accurate